config/config_util.go

```diff
@@ -80,3 +87,15 @@ func mergeConfigItems(dstConf, newConf reflect.Value, fieldPath string) (accepte
 	}
 	return
 }
+
+func atomicWriteConfig(c *Config, confPath string) (err error) {
```

Comment:
What will happen when `confPath` is empty?

Reply:
It will cause an error when try to `Rename` the temporal file to an empty path.